datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/KafkaBasedConnectorConfig.java

```diff
@@ -72,6 +75,8 @@
   private final long _nonGoodStateThresholdMillis;
   private final boolean _enablePartitionAssignment;
 
+  private final String _offsetCheckpointTrackingStrategy;
```

Comment:
Can you save the object instead of the class name? If you create a factory, you may be able to do it.